.ci/update-gherkin-specs.yml

```diff
@@ -0,0 +1,115 @@
+name: update-gherkin-specs
```

Comment:
It seems we could use the proposal for `updatecli.d` to benefit from the vscode integration:
- https://github.com/updatecli/website/blob/83c62785e57908d19db621574b74e1b1aebaf8ad/layouts/shortcodes/ide_jsonschema.html#L7

Reply:
https://github.com/elastic/apm-agent-java/pull/3053/commits/8b4db4c12c3d8d191836ce682938caf7f3e0ee51